src/molecules/Pagination.jsx

```diff
@@ -99,7 +100,12 @@ const Pagination = (props) => {
     return "";
   }
   const getUrl = (pageNumber) => {
-    return baseUrl + "page=" + pageNumber;
+    return (
+      baseUrl +
+      "?page=" +
```

Comment:
This is going to break other uses of paginator, as previously when used with a query string, the query sting is put before the page.